packages/hippy-vue/src/runtime/index.js

```diff
@@ -259,6 +261,16 @@ if (config.devtools && devtools) {
   devtools.emit('init', Vue);
 }
 
+Vue.config._setBeforeRenderToNative = (hook, version) => {
+  if (isFunction(hook)) {
+    if (BEFORE_RENDER_TO_NATIVE_HOOK_VERSION === version) {
+      setBeforeRenderToNative(hook);
+    } else {
+      console.warn('ElementNode API had changed, will ignore beforeRenderToNative hook!');
```

Comment:
console.error('_setBeforeRenderToNative  API had changed, the hook function  will be ignored!');